src/Snapshot/ObjectSnapshot.php

```diff
@@ -41,18 +41,14 @@ public function __construct($object)
 
         $this->raw = $object;
         $this->oid = spl_object_hash($object);
-        $refl      = new ReflectionObject($object);
 
-        if (method_exists('\\ReflectionObject', 'isCloneable') && $refl->isCloneable()) {
-            $this->raw = clone $object;
-        }
+        $export = (array) $object;
+        $class = get_class($object);
 
-        /** @var \ReflectionProperty $reflProperty */
-        foreach ($refl->getProperties() as $reflProperty) {
-            $reflProperty->setAccessible(true);
-            $value = $reflProperty->getValue($object);
+        foreach ($export as $property => $value) {
+            $property = str_replace(["\x00*\x00", "\x00{$class}\x00"], '', $property); // not accessible properties
```

Comment:
and some other things can't remember which too (I think it was in the var_dumper component of symfony)